Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -1079,6 +1079,14 @@ extension SourceKitLSPServer {
       ? nil
       : ExecuteCommandOptions(commands: builtinSwiftCommands)
 
+    var experimentalCapabilties: [String: LSPAny] = [
+      "workspace/tests": .dictionary(["version": .int(2)]),
+      "textDocument/tests": .dictionary(["version": .int(2)]),
+    ]
+    if self.options.backgroundIndexingOrDefault {
+      experimentalCapabilties["workspace/triggerReindex"] = .dictionary(["version": .int(1)])
+    }
```

Comment:
Background indexing might be disabled globally but could be enabled on a per-project level using a `.sourcekit-lsp` file at the project’s root (https://github.com/swiftlang/sourcekit-lsp/blob/main/Documentation/Configuration%20File.md). Thus, I think we should always report the `workspace/triggerReindex` capability. If the project has background indexing disabled, it will just not do anything.

Reply:
Good call, done